src/App.tsx

```diff
@@ -1,7 +1,39 @@
 import React from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+export class App extends React.Component {
+  state = {
+    pressedKey: null,
+  };
+
+  componentDidMount(): void {
+    document.addEventListener('keyup', this.handleKeyUp);
+  }
+
+  componentWillUnmount(): void {
+    document.removeEventListener('keyup', this.handleKeyUp);
+  }
+
+  handleKeyUp = (event: KeyboardEvent) => {
+    this.setState({ pressedKey: event.key });
+  };
+
+  render() {
+    const { pressedKey } = this.state;
+
+    let message;
+
+    if (pressedKey) {
+      message = `The last pressed key is [${pressedKey}]`;
+    } else {
+      message = 'Nothing was pressed yet';
+    }
```

Comment:
It can be written a little shorter:
```suggestion
    let message = 'Nothing was pressed yet';

    if (pressedKey) {
      message = `The last pressed key is [${pressedKey}]`;
    }
```